src/net_processing.cpp

```diff
@@ -2516,20 +2517,22 @@ bool static ProcessMessage(CNode* pfrom, const std::string& strCommand, CDataStr
         bool fMissingInputs = false;
         CValidationState state;
 
-        if (!AlreadyHave(inv) && AcceptToMemoryPool(mempool, state, ptx, true, &fMissingInputs)) {
+        if (!AlreadyHave(inv) && AcceptToMemoryPool(mempool, state, ptx, &fMissingInputs /* pfMissingInputs */,
+                false /* bypass_limits */, 0 /* nAbsurdFee */)) {
             // Process custom txes, this changes AlreadyHave to "true"
             if (nInvType == MSG_DSTX) {
                 LogPrint(BCLog::PRIVATESEND, "DSTX -- Masternode transaction accepted, txid=%s, peer=%d\n",
-                        tx.GetHash().ToString(), pfrom->GetId());
+                         tx.GetHash().ToString(), pfrom->GetId());
                 CPrivateSend::AddDSTX(dstx);
             }
 
             mempool.check(pcoinsTip.get());
             connman->RelayTransaction(tx);
+
             for (unsigned int i = 0; i < tx.vout.size(); i++) {
                 auto it_by_prev = mapOrphanTransactionsByPrev.find(COutPoint(inv.hash, i));
                 if (it_by_prev != mapOrphanTransactionsByPrev.end()) {
-                    for (const auto& elem : it_by_prev->second) {
+                    for (const auto &elem : it_by_prev->second) {
```

Comment:
Why? This breaks dev styling guide `Align pointers and references to the left i.e. use type& var and not type &var.` https://github.com/dashpay/dash/blob/master/doc/developer-notes.md (not to mention that this and changes to whitespaces are unrelated and do not exist in the original PR).

Reply:
was this present before the squash? (I didn't actively do this, CLion potentially did it without me realizing)

Reply:
applied patch and squashed into 11309